Cargo.lock

```diff
@@ -718,6 +787,23 @@ dependencies = [
  "log 0.4.1 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[[package]]
+name = "tokio-proto"
```

Comment:
Please add the `default-features = false` to your `hyper` dependency so that tokio-proto doesn't get pulled in. (Note that tokio-proto seems to be responsible for most of the other unfortunate addeded dependencies like rand 0.3 and slab 0.3.)

conduit-proxy itself uses this:

```toml
hyper = { version = "0.11.22", default-features = false, features = ["compat"] }`
```